landoscript/src/landoscript/util/diffs.py

```diff
@@ -19,10 +21,11 @@ def diff_contents(orig, modified, file):
         # modified does not exist yet; ie: it will be added
         modified_contents = ""
         tofile = "/dev/null"
+        add_remove_metadata = "deleted file mode 100644\n"
```

Comment:
I guess in theory files might have a different mode, but in practice this is all just l10n info so unlikely to be executable?

Reply:
Yeah. I suppose this could be more robust by looking at the file modes at the same time as their contents...probably as a future enhancement though? (I did test, and `git apply` will bail if the mode does not match, so we'll notice if this causes problems...)